wandb/deploys/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt-get update && apt-get upgrade && apt-get install
```

Comment:
Don't need this line anymore.

Reply:
@zythosec indicates this was good practice